docs/administration/spaces/private-space.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Private Space
+description: Information on Private Spaces
+position: 30
+version: 2022.2
```

Comment:
FYI - No harm in leaving this in, but I don't think it prevents the page from showing or anything like that. This used to be for when we displayed multiple versions of the docs, which we no longer do.

Reply:
Ah ok thanks for the info. I saw it at the top of spaces/index.md and thought I'd better include one :)

Reply:
Sorry my bad, the title, description and position are required. It was just `version` that can be removed (or kept, it doesn't hurt)

Reply:
Oh that makes sense! Thanks